e2e/custom_registry/WORKSPACE

```diff
@@ -27,7 +27,7 @@ load("@rules_oci//oci:pull.bzl", "oci_pull")
 
 oci_pull(
     name = "debian",
-    image = "index.docker.io/library/debian",
+    image = "https://index.docker.io/library/debian",
```

Comment:
where do users typically get this string? I think we should optimize for easy copy-paste.

For that matter, I've been doing some rules_docker migrations, and they have separate `registry` and `repository` attributes. Maybe we should optimize for the migration case, if that API was just as good

Reply:
agreed to have separate attributes on the underlying repository rule, but give a wrapper macro with syntax sugar for various shortforms of `image`
can be in a followup PR